scalding-core/src/main/scala/com/twitter/scalding/ExecutionContext.scala

```diff
@@ -91,6 +91,23 @@ trait ExecutionContext {
         case _: HadoopMode =>
           config.get(Config.ReducerEstimators)
             .foreach(_ => flow.setFlowStepStrategy(ReducerEstimatorStepStrategy))
+
+          config.get(Config.FlowListeners).foreach { clsNames: String =>
+            println(s"XXX adding FlowListeners $clsNames")
```

Comment:
we should use proper logging here (same as cascading, not sure which framework they use).